src/wagtail_personalisation/models.py

```diff
@@ -74,11 +73,15 @@ def get_rules(self):
         return segment_rules
 
 
-class PersonalisablePage(Page):
+class AbstractPersonalisablePage(models.Model):
```

Comment:
Can we rename this class to PersonaliablePageMixin ? This will make it more clear that this class should be used as a mixin instead